eng/Versions.props

```diff
@@ -3,8 +3,8 @@
     <MajorVersion>10</MajorVersion>
     <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
-    <PreReleaseVersionLabel>alpha</PreReleaseVersionLabel>
     <PreReleaseVersionIteration>1</PreReleaseVersionIteration>
```

Comment:
You should remove the iteration too (it won't get included since the version label is empty)

Reply:
Fixed in https://github.com/dotnet/source-build-reference-packages/pull/1086/commits/546f721a8c7c3f16dfb0edf489955c4d47e6eb61